packages/web-components/src/drawer-body/drawer-body.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import { spacingHorizontalM, spacingVerticalL } from '../theme/design-tokens.js';
+
+/** Drawer styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+  :host {
+    height: 100%;
+    padding: var(--spacingHorizontalXL);
+    max-height: calc(100vh - (2 * var(--spacingVerticalXXL)));
+  }
+  .drawer-body {
+    display: grid;
+    grid-template-rows: min-content auto min-content;
+    gap: ${spacingVerticalL};
+    overflow-y: auto;
+    max-height: calc(100vh - 2 * ${spacingVerticalL});
+  }
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .title {
+    font-weight: var(--fontWeightSemibold);
+    font-size: var(--fontSizeBase500);
+    line-height: var(--lineHeightBase500);
```

Comment:
We have CSS partials which can be interpolated here - can one of those values be applied here?